azure-pipelines.yml

```diff
@@ -36,16 +36,16 @@ jobs:
       docker pull --quiet rpcs3/rpcs3-travis-xenial:1.6
       docker run                      \
         -v $(pwd):/rpcs3              \
-        --env-file .ci/travis.env \
+        --env-file .ci/ci.env \
```

Comment:
This is pretty close to what I would've done 👍.
Minor nit, the env file is only relevant to docker, so I think it would instead make sense to call the file docker.env, or something along those lines.